controller/codebase.go

```diff
@@ -187,13 +187,7 @@ func (c *CodebaseController) Open(ctx *app.OpenCodebaseContext) error {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
 	}
 	cheClient := che.NewStarterClient(c.config.GetCheStarterURL(), c.config.GetOpenshiftTenantMasterURL(), getNamespace(ctx))
-	workspace := che.WorkspaceRequest{
-		Name:       ctx.WorkspaceID,
-		Repository: cb.URL,
-		Branch:     "master",
-		StackID:    cb.StackID,
-	}
-	workspaceResp, err := cheClient.CreateWorkspace(ctx, workspace)
+	workspaceResp, err := cheClient.StartExistingWorkspace(ctx, ctx.WorkspaceID)
```

Comment:
Just for the note, now the implementation is the following - if there is a name parameter in request body of `POST workspace/` other parameter like repo, branch, stack are simply not taken into account https://github.com/redhat-developer/che-starter/blob/master/src/main/java/io/fabric8/che/starter/controller/WorkspaceController.java#L205

This was the main reason for introducing separate PATCH endpoint for starting existing workspace